shell/components/form/LabeledSelect.vue

```diff
@@ -152,6 +153,9 @@ export default {
   },
 
   methods: {
+    generateRandomAlphaString(length) {
+      return Array.from({ length }, () => String.fromCharCode(97 + Math.random() * 26 | 0)).join('');
+    },
```

Comment:
Should this be a utility function?

Reply:
ah yes... this was the first PR that introduced this method but I never got around to fix it like I did in subsequent PRs. Thanks